themes/custom/droopler_theme/scss/components/_slick-theme.scss

```diff
@@ -43,7 +43,7 @@ $slick-opacity-not-active: 0.25 !default;
 
 .slick-list {
     .slick-loading & {
-        background: #fff slick-image-url("ajax-loader.gif") center center no-repeat;
+        background: #fff url("/libraries/slick-carousel/slick/ajax-loader.gif") center center no-repeat;
```

Comment:
Nie do końca o to chodziło:
1) Dodaj tu zmienną do path `background-image: url('#{$assets-path}/ajax-loader.gif');` i wrzuć tego giga do "images" w skórce. Chodzi o to, że katalog /libraries znajduje się w różnych miejscach w zależności od sposobu instalacji.
2) Powtórz to samo dla innych obrazków z scss, które nie mają `$assets-path` w ścieżce